### PR TITLE
Use a change log.

### LINKED ISSUE #145: README latest versions go out of date easily
It's probably not a good idea to list a latest version in the README:

1. the information goes out-of-date very easily
2. a version number like `1.5.2.0` cannot be used in ARM templates or CLI. API allows only `major.minor` precision. so it's almost useless.

therefore I suggest listing versions  only in Changelog for information purposes.

> image

### LINKED ISSUE #147: [CustomScript] changelog update missing
current changelog mentions only 1.5.1.0 however there's 1.5.2.0 out.

https://github.com/Azure/azure-linux-extensions/blob/e915617d02030b032ff6ae61b5bd00f080296b2c/CustomScript/README.md

### PR DESCRIPTION
**Pull request by boumenot**
Move all version information out of README.md because it becomes stale
quickly.

Closes #145.
Closes #147.

### CODE REVIEW

**Comment by ahmetalpbalkan** [comment]
LGTM, going forward it would be great to refer to the gh issues in the changelog if they are fixing issues as well. Example: https://github.com/Azure/azure-docker-extension#changelog

File: CustomScript/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## vNext (yyyy-mm-dd)
+- Error message misleading [GH-150]
```
Comment (ahmetalpbalkan) [inline CustomScript/CHANGELOG.md]: if you use #NN format directly you will get links to issues for free.

**Comment by ahmetalpbalkan** [comment]
LGTM

File: CustomScript/CHANGELOG.md (hunk 1 above)
Comment (boumenot) [inline CustomScript/CHANGELOG.md]: That's what I want.  I'll fix.

**Pull request merged**